Source/WebCore/Modules/webcodecs/WebCodecsVideoFrameData.h

```diff
@@ -27,16 +27,16 @@
 
 #if ENABLE(WEB_CODECS)
 
-#include "VideoFrame.h"
 #include "VideoPixelFormat.h"
+#include <pal/platform/VideoFrame.h>
 
 namespace WebCore {
 
 struct WebCodecsVideoFrameData {
     // We might want to make memory cost take into account the video frame format.
     size_t memoryCost() const { return 4 * codedWidth * codedHeight; }
 
-    RefPtr<VideoFrame> internalFrame;
+    RefPtr<PAL::VideoFrame> internalFrame;
```

Comment:
This shouldn't be needed. There isn't any PAL::VideoFrame instances that aren't WebCore::VideoFrame instances.